docs/DISTROS.md

```diff
@@ -0,0 +1,170 @@
+# curl distros
+
+<!-- markdown-link-check-disable -->
+
+Lots of organizations distribute curl packages to end users. This is a
+collection of pointers to where to learn more about curl on and with each
+distro.
+
+We discuss curl distro issues, patches and collaboration on the [curl-distros
+mailing list](https://lists.haxx.se/mailman/listinfo/curl-distros).
+
+# Alpine Linux
+
+- curl: https://pkgs.alpinelinux.org/package/edge/main/x86_64/curl
+- curl issues: https://gitlab.alpinelinux.org/alpine/aports/-/issues
+- curl security: https://security.alpinelinux.org/srcpkg/curl
+- curl package source and patches: https://gitlab.alpinelinux.org/alpine/aports/-/tree/master/main/curl
+
+## Arch Linux
+
+- curl: https://archlinux.org/packages/core/x86_64/curl/
+- curl issues: https://gitlab.archlinux.org/archlinux/packaging/packages/curl/-/issues
+- curl security: https://security.archlinux.org/package/curl
+- curl wiki: https://wiki.archlinux.org/title/CURL
+
+# Clear Linux
+
+- curl: https://github.com/clearlinux-pkgs/curl
+- curl issues: https://github.com/clearlinux/distribution/issues
+
+# Conary
+
+- curl: https://github.com/conan-io/conan-center-index/tree/master/recipes/libcurl
+- curl issues: https://github.com/conan-io/conan-center-index/issues
+- curl patches: https://github.com/conan-io/conan-center-index/tree/master/recipes/libcurl (in `all/patches/*`, if any)
+
+# conda-forge
+
+- curl: https://github.com/conda-forge/curl-feedstock
+- curl issues: https://github.com/conda-forge/curl-feedstock/issues
+
+## curl-for-win
+
+(this is the official curl binaries for Windows shipped by the curl project)
+
+- curl: https://curl.se/windows/
+
+Issues and patches for this are managed in the main curl project.
+
+## Debian
+
+- curl: https://tracker.debian.org/pkg/curl
+- curl issues: https://bugs.debian.org/cgi-bin/pkgreport.cgi?pkg=curl
+- curl patches: https://udd.debian.org/patches.cgi?src=curl
```

Comment:
- curl patches: https://salsa.debian.org/debian/curl (in `debian/*` branches, inside the folder `debian/patches`)

Reply:
To clarify, UDD won't list the patches for the older Debian releases, such as buster or stretch, while we do keep all of the packaging in a single git repo.